csharp/src/SeedLang/Interpreter/VM.cs

```diff
@@ -32,45 +32,51 @@ private enum State {
       Stopped,
     }
 
-    private readonly Sys _sys = new Sys();
-    private readonly Registers _registers = new Registers();
-    private readonly VisualizerCenter _visualizerCenter;
-
-    public GlobalEnvironment Env { get; } = new GlobalEnvironment(NativeFunctions.Funcs.Values);
+    // The standard output that is used for print() and expression statements.
+    public TextWriter Stdout { get; private set; } = Console.Out;
 
     public bool IsRunning => _state == State.Running;
     public bool IsPaused => _state == State.Paused;
     public bool IsStopped => _state == State.Stopped;
 
+    private readonly VisualizerCenter _visualizerCenter;
+    private readonly Registers _registers = new Registers();
+
     private State _state = State.Stopped;
 
+    private Module _module;
     private CallStack _callStack;
     private Chunk _chunk;
     private int _pc;
 
-    // The hash table to store defined global variable names.
-    private HashSet<string> _globals;
+    // Returns the names in the module namespace, if the given value is a module object. Otherwise
+    // returns an empty list.
+    internal static VMValue Dir(VMValue value) {
```

Comment:
"Dir" is a bit confusing with the concept "directory" as in a filesystem.

Is "ModuleDir" better here?